### DISCUSSION TITLE
Regarding stable release

### DISCUSSION THREAD

**Discussion by deoramanas** [Netflix/conductor#2716] [General]
I have done a bugfix https://github.com/Netflix/conductor/issues/2678.
Please let me know in which stable release(v3+) it'll be available??

@aravindanr  @apanicker-nflx

**Comment by jxu-nflx** [comment]
Hello @deoramanas we just released the fix along with v3.4.2: https://github.com/Netflix/conductor/releases/tag/v3.4.2
